recipes/carpedeam/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "1.0.0" %}
+{% set sha256 = "92deec3d670810f8433c0345758588b47fa460349e61ce8aa549374f5cf4a812" %}
+
+package:
+  name: carpedeam 
+  version: {{ version|replace("-", ".")  }}
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('carpedeam', max_pin=None) }}
```

Comment:
Are you sure about `None` here ?
Please see the PR [template](https://github.com/bioconda/bioconda-recipes/blob/master/.github/PULL_REQUEST_TEMPLATE.md) what are the possible values and their meaning.

Reply:
Thanks for the reference, I think `None` should be fine, but to be safe I changed to `"1.0"`